templates/eslint/src/fixers/openai-worker.ts

```diff
@@ -156,9 +174,11 @@ runAsWorker(async (codeToComment: string): Promise<string | null> => {
       { role: "assistant", content: sampleResponse },
       { role: "user", content: samplePromptTwo },
       { role: "assistant", content: sampleResponseTwo },
+      { role: "user", content: samplePromptThree },
+      { role: "assistant", content: sampleResponseThree },
       { role: "user", content: codeToComment },
     ],
-    model: "gpt-4o",
+    model: "gpt-3.5-turbo",
```

Comment:
gonna use 3.5-turbo because it's WAY cheaper. if we end up with poor quality results though, I'll bump it back up to 4o

Reply:
ugh. it's so much slower though

Reply:
oh it's also just wrong. reverting in the next PR

Reply:
switched to 4o here: https://github.com/alchemyplatform/aa-sdk/pull/735